pkg/operator/controller/ingress/load_balancer_service.go

```diff
@@ -666,6 +681,9 @@ func shouldRecreateLoadBalancer(current, desired *corev1.Service, platform *conf
 	if platform.Type == configv1.AWSPlatformType && !serviceEIPAllocationsEqual(current, desired) {
 		return true, "its eipAllocations changed"
 	}
+	if platform.Type == configv1.OpenStackPlatformType && !serviceLoadBalancerIPsEqual(current, desired) {
+		return true, "its load balancer IP changed"
+	}
```

Comment:
We have not test coverage for `shouldRecreateLoadBalancer`.  As a follow-up, we should add a unit test.

Reply:
Yea I can take that action, but I'm removing it in my PR https://github.com/openshift/cluster-ingress-operator/pull/1142/commits/c5ecad99f07ee972bed670cb3a67400196f0e6f9 actually.

Reply:
I've added it in an extra commit :+1: